### PR TITLE
Update first_look_at_the_editor.rst

### PR DESCRIPTION
**Pull request by CyberSkull**
Updating macOS shortcuts and cleaning up the click shortcut.
* Clarified clicking on keywords help function shortcuts.

### CODE REVIEW

File: getting_started/introduction/first_look_at_the_editor.rst [hunk 1]
```diff
@@ -144,11 +144,10 @@ Godot comes with a built-in class reference.
 You can search for information about a class, method, property, constant, or
 signal by any one of the following methods:
 
-* Pressing :kbd:`F1` (or :kbd:`Opt + Space` on macOS, or :kbd:`fn + F1` for laptops with a :kbd:`fn` key) anywhere in the editor.
+* Pressing :kbd:`F1` (or :kbd:`⌥ Option + Space` on macOS, or :kbd:`fn + F1` for laptops with a :kbd:`fn` key) anywhere in the editor.
```
Comment (skyace65) [inline getting_started/introduction/first_look_at_the_editor.rst:147]:
```suggestion
* Pressing :kbd:`F1` (or :kbd:`Opt + Space` on macOS, or :kbd:`fn + F1` for laptops with a :kbd:`fn` key) anywhere in the editor.
```

File: getting_started/introduction/first_look_at_the_editor.rst [hunk 2]
```diff
@@ -144,11 +144,10 @@ Godot comes with a built-in class reference.
...
 * Clicking the "Search Help" button in the top-right of the Script main screen.
 * Clicking on the Help menu and Search Help.
-* Clicking while pressing the :kbd:`Ctrl` key on a class name, function name,
-  or built-in variable in the script editor.
+* :kbd:`Ctrl + Click` (:kbd:`⌘ Command + Click` on macOS) on a class name, function name, or built-in variable in the script editor.
```
Comment (skyace65) [inline getting_started/introduction/first_look_at_the_editor.rst:150]:
```suggestion
* :kbd:`Ctrl + Click` (:kbd:`Cmd + Click` on macOS) on a class name, function name, or built-in variable in the script editor.
```

**Review by skyace65 — CHANGES_REQUESTED** [review]
Same thing as my review on your other PR. Please keep option and command shortened, and we don't want to do symbols for these shortcuts.

**Comment by CyberSkull** [comment]
Thanks for all your feedback!

**Pull request merged**